docs/UserGuide.md

```diff
@@ -58,7 +58,7 @@ We hope that this guide is helpful in smoothing out the learning curve for you!
 
 --------------------------------------------------------------------------------------------------------------------
 
-## 3. Terminologies
+## 3. Product Overview
```

Comment:
Hmm, i am afraid user might mixed up the definition of the product here and the product in myInsuRec. Perhaps name it `Overview` or `Application Overview` to separate them apart

Reply:
will change to overview